src/Dafda/Dafda.csproj

```diff
@@ -7,7 +7,7 @@
     <Description>A small client for Kafka</Description>
     <RepositoryUrl>https://github.com/dfds/dafda</RepositoryUrl>
     <PackageProjectUrl>https://github.com/dfds/dafda</PackageProjectUrl>
-    <Version>0.12.4</Version>
+    <Version>0.12.2</Version>
```

Comment:
just keep it as it it. we will roll forward.

Reply:
I will unlist 0.12.4 from our nuget feed when 12.5 is out

Reply:
I changed it to 0.12.4